NICE.Identity.Management/ClientApp/src/views/UsersList/__tests__/UsersList.test.js

```diff
@@ -138,8 +159,28 @@ describe("UsersList", () => {
 		});
 	});
 
+	it("should filter users to those who have access to a specific service when radio button is clicked", async () => {
```

Comment:
it's not clear to me what this is testing. is "3" a website id?

Reply:
Hi John, it's still not clear to me what's going on with that function. it looks like you've changed the code to:

wrapper.find("#filter_dev_3").simulate("change");

what's filter_dev_3?

also, the test description says service. i thought we were filtering by website now?

Reply:
Hi John, the id is generated by the filter panel stuff from the design system. It names the id according to the heading used and the value of the checkbox. In the dummy services.json file there is a website with id of '3' and environment name 'Dev'. The test simulates checking this box and then tests that it brings back 2 users, which it gets from the dummy users.json file.